packages/component/src/Styles/StyleSet/SendBoxTextArea.js

```diff
@@ -26,9 +26,10 @@ export default function createSendBoxTextAreaStyle({
         fontFamily: 'inherit',
         fontSize: 'inherit',
         height: '100%',
-        overflowY: 'scroll',
+        overflowY: 'auto',
         whiteSpace: 'pre-wrap',
         width: 'inherit',
+        wordBreak: 'break-word'
```

Comment:
Should we following `styleOptions.messageActivityWordBreak`? @corinagum what do you think?

If yes, should we add more tests?

Reply:
hmmm good point. However, if it's not 'break-word' it creates undesired behavior so I think we would want it to stay on 'break-word' always. 

Thoughts?

Reply:
My brain stuck last week. Please keep `break-word`.